app/views/AdminView/components/Appendices/columnDefs.ts

```diff
@@ -0,0 +1,36 @@
+import { formatDate } from '@/utils/date';
+import { ColDef } from '@ag-grid-community/core';
+
+const columnDefs: ColDef[] = [
```

Comment:
non-blocking, but since there's only a few columns so this isn't a massive array, I think it would be better just in the same file as the table